nltk/tag/stanford.py

```diff
@@ -204,6 +205,79 @@ def parse_output(self, text, sentences):
 
         raise NotImplementedError
 
+class CoreNLPTagger(CoreNLPParser, TaggerI):
+    def __init__(self, url='http://localhost:9000', encoding='utf8'):
```

Comment:
it might make sense to take *args, **kwargs that are passed to super.

Similarly below.